src/net/http/ToStream.cpp

```diff
@@ -66,6 +64,8 @@ DownloadToStream(CurlGlobal &curl, CurlEasy easy,
 
   AtScopeExit(&env) { env.SetCancelHandler({}); };
 
+  const Net::ProgressAdapter progress(request.GetEasy(), env);
```

Comment:
This does indeed fix the problem, but you didn't understand why (at least your commit message doesn't explain it). This is why, and it's a better solution: https://github.com/XCSoar/XCSoar/commit/d5457d3fd35094c754cf7e55addb378453eb6cf7

Reply:
It was only a quick and dirty fix ;-(
Solved